lerna.json

```diff
@@ -7,6 +7,9 @@
   "command": {
     "bootstrap": {
       "forceLocal": true
+    },
+    "publish": {
+      "registry": "https://wombat-dressing-room.appspot.com"
```

Comment:
Does this mean we don't have to include this in every package.json?

Reply:
I am not sure. This needed to be added so that I could log in via the command line.